themes/beaver/assets/css/3082-layout.css

```diff
@@ -940,7 +940,7 @@ img.mfp-img {
 }
 
 .fl-node-ec25np07hf63 > .fl-row-content-wrap {
-
+  background-image: url(/wp-content/uploads/2022/10/white-border-background.webp);
```

Comment:
Please render I deleted it

Reply:
no need, I used it only for 404.css, 590-layout.css

Reply:
I deleted those files. You can just delete this line